dsgrid/config/project_config.py

```diff
@@ -131,6 +134,7 @@ class InputDatasetModel(DSGBaseModel):
             "The version string must be in semver format (e.g., '1.0.0') and it must be a valid/"
             "existing version in the registry.",
         ),
+        updateable=False,
```

Comment:
Seems like the version would be updateable?

Reply:
Correct. Fixed.